src/NameFieldSet.tsx

```diff
@@ -59,6 +59,32 @@ export class NamesFieldSet extends React.Component<INameFieldSetProps, INameFiel
     this.setState({ names: [...this.state.names, ""] });
   }
 
+  private arrayContainsDuplicates = (a: string[]) => {
```

Comment:
change variable name to a more descriptive name. Something like names or nameArray